frontend/src/toolbar/elements/heatmapLogic.ts

```diff
@@ -50,10 +57,12 @@ export const heatmapLogic = kea<heatmapLogicType<ElementsEventType, CountedHTMLE
             {
                 resetEvents: () => [],
                 getEvents: async ({ $current_url }: { $current_url: string }, breakpoint) => {
-                    const params = {
+                    const params: Record<string, any> = {
                         properties: [{ key: '$current_url', value: $current_url }],
                         temporary_token: toolbarLogic.values.temporaryToken,
+                        ...heatmapLogic.values.heatmapFilter,
```

Comment:
Wouldn't it be more performant/nicer/something (I'm not sure, I have not engineered Kea) as 
```TypeScript
loaders: ({ values }) => {
    …
    values.heatmapFilter,
    …
}
```
?

Reply:
It's the same, but uncommon, so done!